<launch>
    <!-- Start Gazebo server -->
    <node name="gazebo" pkg="gazebo_ros" type="gzserver" args="-s libgazebo_ros_init.so -s libgazebo_ros_factory.so --verbose $(find my_sim_pkg)/worlds/pizza_delivery_world.sdf" output="screen">
        <param name="use_sim_time" value="true"/>
    </node>

    <!-- Start Gazebo client -->
    <node name="gazebo_gui" pkg="gazebo_ros" type="gzclient" output="screen"/>

    <!-- Spawn entities dynamically -->
    <node pkg="my_sim_pkg" type="spawn_entities.py" name="spawn_entities" output="screen"/>
</launch>
